### ISSUE TITLE
Breakout column missing in mongo questions

### ISSUE THREAD

**Issue by salsakran** [metabase/metabase#2814]
Count by a category field breakout isn't returning the breakout field itself--

Metabase is running on Elasticbeanstalk (AWS) and I have connected MongoDB (3.2.7) with a Chrome Browser

From 
http://discourse.metabase.com/t/category-missing-but-visible-in-raw-data/478/1

```
If I view Raw data everything looks great, but when I try to see bar chart for
 Categories it only gives me category-wise numbers (category names are 
missing). Please view below screenshot -

Metabase 1.JPG1847x859 47.9 KB
I tried creating bar charts with existing sample data and everything looks great. 
Only when am using MongoDB, category names (or other fields) are missing.

Please assist.

```
![](http://discourse.metabase.com/uploads/default/original/1X/c190a649eed1c6124beea7f0da1f092317f29f1e.JPG)

![](http://discourse.metabase.com/uploads/default/original/1X/3c3682ba435bb564d198717a5355d654e0276e2d.JPG)

```
Nope, there's nothing strange about Categories (text field but it does 
include '/' & '-' in some values)

I am using Chrome but this doesn't seem to be browser related issue. 
I am getting the same error when I am connecting Mongo DB through
 'Metabase on Mac' on my macbook.

Please find console errors logged -
```

![](http://discourse.metabase.com/uploads/default/original/1X/9b632e7cc38f23ccd49b5cf8b7e220bedf004219.JPG)

**Comment by camsaul** [comment]
I haven't been able to reproduce this on 0.18.0 or 0.18.1. I tried with a couple different Mongo DBs. The screenshots are definitely from a pre-0.18.0 version of Metabase so this may have been inadvertently fixed in a recent release

**Comment by camsaul** [comment]
It looks like this bug was inadvertently fixed in 0.18. I tried reaching out to the guy who brought the issue up of Discourse but haven't heard back for a few months now. 

I vote that we close this as Unable to Reproduce, and if anyone else sees it we can open a new issue

**Comment by salsakran** [comment]
👍
